setup.py

```diff
@@ -71,6 +71,11 @@ def get_version():
             "pyvisa-py==0.5.3",
             "qm-qua>=1.0.1",
             "qualang-tools>=0.13.1",
+            "zhinst-toolkit==0.5.0",
+            "laboneq==2.1.0",  # 2.2.0
```

Comment:
Which version do you need?

If it's just greater than 2.1.0, and smaller than 2.3.0, write:
```suggestion
            "laboneq>=2.1.0,<2.3.0",
```
If instead you really need 2.1.0, and you just tried 2.2.0 but it is not working yet, you could replace 2.3.0 in this suggestion with 2.2.0, or just drop the comment.

Reply:
It is exactly what you said about it not working yet with 2.2.0. It works now and data seems right so i will change it to laboneq>=2.2.0